app/src/main/java/ar/com/wolox/android/example/ui/auth/login/AuthFragment.kt

```diff
@@ -54,6 +55,17 @@ class AuthFragment @Inject constructor() : WolmoFragment<FragmentLoginBinding, A
 
     override fun openBrowser(url: String) = requireContext().openBrowser(url)
 
+    override fun showErrorLogin(status: ResponseStatus) {
+        when (status) {
+            ResponseStatus.ERROR_CREDENTIALS -> toastFactory.show(R.string.request_error_login)
+            else -> toastFactory.show(R.string.default_error)
+        }
+    }
+
+    override fun showLoader(visible: Boolean) {
+        with(binding) { loaderLogin.visibility = if (visible) View.VISIBLE else View.INVISIBLE }
```

Comment:
I think in this cases where you only access to one element of the viewBinding, it's more easy to write `binding.loaderLogin`

Reply:
You can also use the `.togglePresence() `method. It's a View extension function.
`loaderLogin.togglePresence(visible)`

Reply:
Also, you could replace 
loaderLogin.visibility = if (visible) View.VISIBLE else View.INVISIBLE
by
loaderLogin.toggleVisibility(visible)